rllib/BUILD

```diff
@@ -1861,6 +1868,13 @@ py_test(
     srcs = ["policy/tests/test_compute_log_likelihoods.py"]
 )
 
+py_test(
```

Comment:
Tests the new individual Policy checkpointing and native model saving mechanisms.